README.md

```diff
@@ -1,2 +1,149 @@
-# Yoga-Class-Refactored
-A Refactored Yoga Application
+# About
+- Backend application demo using Spring Boot
+- Application to manage Yoga classes
+- Refactored of [old version](https://github.com/web-development-course-way/Yoga-Class)
+# Tech Stack
+- Docker
+- PostgreSQL & pgAdmin 
+- Java & Spring boot
+# Installation
+01. Clone Repository and open project in IDE
+02. [Set up database](https://github.com/web-development-course-way/Yoga-Class-Refactored/tree/enhancement/docs?tab=readme-ov-file#setting-up-database)
+03. Run application
+## Setting up database
+01. Run Docker
+02. Run Docker compose using following command `docker compose up` from the root of the project
+03. Open [pgAdmin](http://localhost:5050/browser/)
```

Comment:
remove hyperlink, since it can run  on different port.

Reply:
It's configured to 5050 in `docker-compose.yml` file,
so why remove it if it will always work? 
considering that anyone trying the project won't play with `docker-compose.yml` file